docs/data/joy/components/autocomplete/AutocompleteHint.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import PropTypes from 'prop-types';
```

Comment:
Can't we just alias `prop-types` to `prop-types-compat` in our docs ?

Reply:
I rather have all the places use the same package instead of aliasing only the docs. To reduce the ambiguity.

Reply:
That was going to be my next question. Why not alias `prop-types` to `prop-types-compat` everywhere ?
For prod builds, I am sure we can replace `prop-types` with `prop-types-compat` in the output with some existing babel plugin.

Reply:
imo, if we do a fork, I'd rather update all imports to the fork. it would be more explicit for maintainers and contributors. they wouldn't need to know about build processes rewriting imports to be able to understand how the code works.